package.json

```diff
@@ -6,10 +6,38 @@
     "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.9",
+    "@babel/cli": "^7.4.4",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/parser": "^7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
+    "@babel/plugin-transform-async-to-generator": "^7.7.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.4.4",
+    "@babel/plugin-transform-object-assign": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.6.2",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.16.0",
+    "@babel/register": "^7.4.4",
+    "@babel/runtime": "^7.7.2",
+    "@babel/runtime-corejs2": "^7.4.5",
+    "@babel/traverse": "^7.5.5",
+    "babel-plugin-dynamic-import-node-babel-7": "^2.0.7",
+    "babel-plugin-formatjs": "10.2.20",
     "commander": "^2.20.0",
+    "eslint": "^7.31.0",
+    "eslint-config-prettier": "4.0.0",
+    "eslint-import-resolver-webpack": "0.10.0",
+    "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-import": "2.23.4",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-prettier": "3.0.1",
+    "eslint-plugin-react": "7.24.0",
+    "prettier": "^1.18.2",
```

Comment:
With the proposed solution, I suppose now we have two sets of eslint we need to maintain, is there a necessity that we keep both of them to be the same?

Reply:
@alexvuong I believe that we want to have multiple setups: one for the monorepo developers and the other one for other developers working on their pwa-kit projects. Even though both setups start out to be nearly identical, but they may deviate more over time. So I think the duplicate code is all right. 

But @bendvc you would know better. Was it intended to be that way?

Reply:
If we change the root eslint config to `{ extends: ['pwa-kit'] }` and add `./packages/eslint-config-pwa-kit` and `./packages/babel-preset-pwa-kit` as dev dependencies then we will only need to maintain the shared config values (internal + external) in a single location.

Reply:
@vmarta Yes, you are correct, that is the intention. That is also what we had prior to this change. You can see that there are eslint configurations in both the "cli" and the "internal build tools". One of the issues with the eslinting before this was that only projects get linted, not the scripts folder that is a sibling folder. I thought that was less than ideal. 

@wjhsf Yes you can do something like that. Do I think it's a little weird to use the output of a package of a mono repo in the root, maybe.